portal/portal-render-engine-impl/pack/src/webapp/vm/morpheus/site.vm

```diff
@@ -139,6 +142,12 @@
                 ${d}PBJQ = jQuery.noConflict(true); // Safely return $ to the tool jQuery
                 if ( ! dollarEmpty ) jQuery = ${d};  // Return jQuery to point at the tool jQuery
             }
+            if (notJQuery) {
```

Comment:
I was thinking of just reusing secondJQuery rather than have a new variable, but wasn't 100% if the jQuery variable was needed to be set to something (though it was used in the console log message).